assets/multiqc_config.yml

```
@@ -4,9 +4,39 @@ report_comment: >
   <a href="https://nf-co.re/rnafusion/dev/docs/output" target="_blank">documentation</a>.
 
 report_section_order:
-  software_versions:
+  nf-core-rnafusion-methods-description:
     order: -1000
-  nf-core-rnafusion-summary:
+  software_versions:
     order: -1001
+  nf-core-rnafusion-summary:
+    order: -1002
 
 export_plots: true
+
+# Run only these modules
+run_modules:
+  - custom_content
+  - fastqc
+  - fastp
+  - star
+  - samtools
+  - picard
+  - arriba
+
+module_order:
+  - fastqc:
+      name: "FastQC (raw)"
+      info: "This section of the report shows FastQC results before adapter trimming."
+      path_filters:
+        - "*.zip"
+  - fastp
```

```suggestion
  - fastp
  - fastqc:
      name: "FastQC (raw)"
      info: "This section of the report shows FastQC results before adapter trimming."
      path_filters:
        - "*.zip"
```
we could move this up just before fastqc so we have both fastqc results one after another. it would seem clearer to me, but up to you!